.github/workflows/tests.yml

```diff
@@ -0,0 +1,44 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        python-version: [3.6, 3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: conda-incubator/setup-miniconda@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      shell: bash -l {0}
+      run: |
+        conda install -c conda-forge rdkit
+        python -m pip install --upgrade pip
+        python -m pip install flake8 pytest parameterized
+        python -m pip install git+https://github.com/bp-kelley/descriptastorus
+        python -m pip install -e .
+    - name: Lint with flake8
+      shell: bash -l {0}
+      run: |
+        # stop the build if there are Python syntax errors or undefined names
+        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+    - name: Test with pytest
+      shell: bash -l {0}
+      run: |
+        pytest
```

Comment:
Do you know if this tests the cloned source code or the pip installed copy?

Reply:
The `actions/checkout@v2` step clones the source code so it should be testing whatever code is pushed to master rather than the pip installed copy.

Reply:
In that case, is `pip install -e .` above necessary? Or is it intended to test whether pip install works?